client/apps/settings/client/curations/gucci/index.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import DropDownItem from 'client/components/drop_down/index.jsx'
+import { SaveButton } from '../components/save_button.jsx'
+import { SectionAdmin } from './components/section.jsx'
+import { SeriesAdmin } from './components/series.jsx'
+
+export class GucciAdmin extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      curation: props.curation,
+      isSaved: true,
+      activeSection: null
+    }
+  }
+
+  setActiveSection = (i) => {
+    i = i === this.state.activeSection ? null : i
+    this.setState({activeSection: i})
+  }
```

Comment:
This is more a suggestion for the future (since we have an insane deadline), but I was hoping to see less repeated code throughout these Admin curations. For example, we've seen [this method](https://github.com/artsy/positron/blob/master/client/apps/settings/client/curations/display/index.jsx#L19-L22) implemented a couple times already. This admin, Venice, and YIA share a lot of functionality that could benefit by having a "shell admin" that gets extended by each of the custom ones. This would be a fun one to pair on!

Reply:
Yeah I think we need a container for the dropdown component -- something similar to how the dropDown list works, where these controls can wrap everything inside.  Venice is actually slightly different because it allows multiple panels to be different. Figured it was more labor than makes sense for this timeline but happy to look at this together soon.